ui/src/app/help/index.tsx

```diff
@@ -1,5 +1,5 @@
 import {Help} from './components/help';
 
 export default {
-    component: Help
+    component: Help,
```

Comment:
Is adding a comma here an intended change?

Reply:
No @wtam2018 , will correct the lint errors.